dashboards/MySQL_Query_Response_Time.json

```diff
@@ -44,6 +45,7 @@
             "type": "dashboards"
         }
     ],
+    "refresh": "1m",
```

Comment:
Yes, we do. Here are the rules that we must follow: https://confluence.percona.com/display/SER/Grafana+dashboards+rules